.travis.yml

```diff
@@ -6,7 +6,6 @@ cache:
 notifications:
   email: false
 node_js:
-  - 'node'
```

Comment:
Doing this opens us up to stuff like this: https://stackoverflow.com/questions/55059748/travis-jest-typeerror-cannot-assign-to-read-only-property-symbolsymbol-tostr

I think the benefit of testing an unstable version of node isn't worth stuff like this.